draft-ietf-jsonpath-base.md

```diff
@@ -1528,13 +1514,10 @@ A function expression is well-typed if all of the following are true:
   its declared result type.
 * Each argument of the function can be used for the declared type of the corresponding declared
   parameter according to one of the following rules:
-   * The argument is a function expression with declared result type that is the same as the declared type of the parameter.
-   * The argument is a literal primitive value and the defined type of the parameter is `ValueType`.
-   * The argument is a Singular Query or `filter-query` (which includes
-     Singular Queries), or a function expression with declared result
-     type `NodesType` and the defined type of the parameter is `NodesType`.
-     Where the declared type of the parameter is
-     not `NodesType`, a conversion applies.
+   * The argument is a function expression with declared result type that is the same as the declared type of the parameter or
+     with declared result type `NodesType` and the declared type of the parameter is `LogicalType` and a conversion applies.
```

Comment:
is "a conversion applies" an additional part of the condition?
I wouldn't think so.

Reply:
Correct. I'm changing the text.